src/iproject.ts

```diff
@@ -136,6 +136,50 @@ export class IProject {
     }
   }
 
+  public async moveUp(pathMovingUp: string){
```

Comment:
Since the `moveUp` and `moveDown` functions are very similar, we can maybe just have one method and pass in the direction the include path is to be moved in